doc/ruleSets/index.rst

```diff
@@ -2,19 +2,19 @@
 List of Available Rule sets
 ===========================
 - `@DoctrineAnnotation <./DoctrineAnnotation.rst>`_
-- `@PER <./PER.rst>`_
 - `@PER:risky <./PERRisky.rst>`_
+- `@PER <./PER.rst>`_
 - `@PHP54Migration <./PHP54Migration.rst>`_
 - `@PHP56Migration:risky <./PHP56MigrationRisky.rst>`_
-- `@PHP70Migration <./PHP70Migration.rst>`_
 - `@PHP70Migration:risky <./PHP70MigrationRisky.rst>`_
-- `@PHP71Migration <./PHP71Migration.rst>`_
+- `@PHP70Migration <./PHP70Migration.rst>`_
 - `@PHP71Migration:risky <./PHP71MigrationRisky.rst>`_
+- `@PHP71Migration <./PHP71Migration.rst>`_
 - `@PHP73Migration <./PHP73Migration.rst>`_
-- `@PHP74Migration <./PHP74Migration.rst>`_
 - `@PHP74Migration:risky <./PHP74MigrationRisky.rst>`_
-- `@PHP80Migration <./PHP80Migration.rst>`_
+- `@PHP74Migration <./PHP74Migration.rst>`_
```

Comment:
I would actually prefer to have `@PHP74Migration` before `@PHP74Migration:risky`

Reply:
Yeah, me too, `Symfony\Component\Finder\Iterator\SortableIterator::SORT_BY_NAME_NATURAL` apparently is not the natural order as in [natsort](https://www.php.net/manual/en/function.natsort.php).